### PR TITLE
Page changes

### PR DESCRIPTION
**Pull request by sarunas**
- Adds `minimized` property to force minimized page state
- Adds `sidePadding` for Page.Content to allow use side padding only for content

## Before you request a review, please make sure to check the following boxes:

- [x] Explain what has changed.
- [x] Needed for New Blog create/edit post page.
- [ ] Added tests.
- [x] Update documentation.
- [x] Merge master with no conflicts.
- [ ] Make sure the build is passing on ci.
- [ ] If this pr closes any issue, please add a github link to close it.
- [ ] If you changed something in the product design/UX, you got the approval of [Ben Benhorin](https://wix.slack.com/messages/@benb).

### CODE REVIEW

File: src/Page/Page.js [hunk 1]
```diff
@@ -185,9 +166,11 @@ class Page extends WixComponent {
                 style={{height: `${headerHeight + (PageTail ? -SCROLL_TOP_THRESHOLD : 39)}px`}}
                 />
           }
-          <div className={s.content} style={pageDimensionsStyle}>
-            {this._safeGetChildren(PageContent)}
-          </div>
+
+          {PageContent && React.cloneElement(PageContent, {
+            maxWidth,
+            sidePadding: getProp(PageContent, 'sidePadding', sidePadding)
```
Comment (argshook) [inline src/Page/Page.js]: i guess you don't need `getProp` if you have `defaultProps` defined

File: src/Page/Page.js [hunk 2]
```diff
@@ -4,26 +4,30 @@ import WixComponent from '../BaseComponents/WixComponent';
...
+    super(props);
 
     this._setContainerScrollTopThreshold(false);
     this._handleScroll = this._handleScroll.bind(this);
 
     this.state = {
       headerHeight: 0,
       tailHeight: 0,
-      minimized: false
+      minimized: Boolean(props.minimized)
```
Comment (argshook) [inline src/Page/Page.js]: missing `PropTypes`. also, if you have `defaultProp`, no need for `Boolean`

File: stories/Page/FullPageExample.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 
-import Page from 'wix-style-react/Page';
-import Button from 'wix-style-react/Button';
+import Page from '../../src/Page';
```
Comment (argshook) [inline stories/Page/FullPageExample.js]: why? imports from `wix-style-react` should work just fine.

File: src/Page/Page.js (hunk 1 above)
Comment (sarunas) [reply src/Page/Page.js]: This getProp, does not replace `sidePadding` on `PageContent` if its there:
```
<Page sidePadding={60}>
  <Page.Content sidePadding={0}>...</Page.Content>
</Page>
```

File: src/Page/Page.js (hunk 2 above)
Comment (sarunas) [reply src/Page/Page.js]: Probably we'll remove this feature - instead will change how of `Page.Header` works without `title`.

File: src/PageHeader/README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ Page.Header component is an implementation for the Header which is nested inside
 | minimized | bool | false | true | This property is being supplied by the Page component, it's value changes by the state of the scrolled content |
 | hasBackgroundImage | bool | false | true | This property is being supplied by the Page component, it's value reflects if the Page has a background image or not |
 | breadcrumbs | Breadcrumbs | null | false | Wix-Style-React Breadcrumbs component |
-| title | string | '' | true | The main title text |
+| title | string | '' | false | The main title text |
```
Comment (argshook) [inline src/PageHeader/README.md]: @amiryonatan why do you have readme, i see autodocs parses this component nicely

Comment (amiryonatan) [reply src/PageHeader/README.md]: Good question, maybe it didn't parse it nicely before ?

Comment (argshook) [reply src/PageHeader/README.md]: i donno. it's this one right? https://wix.github.io/wix-style-react/?selectedKind=10.%20Page&selectedStory=10.3%20Header%20with%20Options&full=0&addons=0&stories=1&panelRight=0

File: src/Page/Page.js [hunk 5]
```diff
@@ -121,6 +121,7 @@ class Page extends WixComponent {
       PageTail
     } = getChildrenObject(children);
 
+    const fullScreen = PageContent && PageContent.props.fullScreen;
```
Comment (amiryonatan) [inline src/Page/Page.js]: contentFullScreen

Comment (amiryonatan) [reply src/Page/Page.js]: Also, PageContent is mandatory u don't need to check it

**Pull request merged**